src/app/common-ui/Popup/index.js

```diff
@@ -4,42 +4,48 @@ import styles from './style.scss';
 import PropTypes from 'prop-types';
 
 import CloseButton from 'assets/images/close-button.png';
-
+import ErrorSign from 'assets/images/error.png';
+import SucessSign from 'assets/images/success.png';
+import WarningSign from 'assets/images/warning.png';
 import cx from 'classnames';
 
+const POPUP = {
+  init: 0,
+  error: 1,
+  success: 2,
+  warning: 3,
+};
+
 function Popup({ show, title, content, buttonText, variant, onButtonClick }) {
```

Comment:
`show` should be a boolean only

Split `show` into `show: boolean` and `type: PopupType` with `enum PopupType { Error, Warning, Success }`

See https://www.typescriptlang.org/docs/handbook/enums.html